cel2sql.go

```diff
@@ -401,6 +401,34 @@ func (con *Converter) callContains(target *exprpb.Expr, args []*exprpb.Expr) err
 	return nil
 }
 
+func (con *Converter) callReplace(target *exprpb.Expr, args []*exprpb.Expr) error {
+	con.str.WriteString("REPLACE(")
+	if target != nil {
+		nested := isBinaryOrTernaryOperator(target)
+		err := con.visitMaybeNested(target, nested)
+		if err != nil {
+			return err
+		}
+		con.str.WriteString(", ")
+	}
+
+	if len(args) != 2 {
+		return fmt.Errorf("string_replace_string_string_int variant of string.replace() is not supported")
+	}
+
+	for i, arg := range args {
+		err := con.Visit(arg)
+		if err != nil {
+			return err
+		}
+		if i < len(args)-1 {
```

Comment:
(Optional) I would personally go for `if i != 0` before the `con.Visit(arg)`

Reply:
Actually, even better option would be to always write the comma before the visit call since we have the `target` handling branch where we can remove the comma writing from

Reply:
great point. I just copy/pasted other "call*" funcs